vinca.yaml

```diff
@@ -158,6 +158,12 @@ packages_select_by_deps:
   # geodesy (required for many outdoors robotics packages)
   - geodesy
 
+  # These are the packages that are not build on wasm32, but are built on
+  # all the other platforms
+  - if: not wasm32
+    then:
+      - mocap4r2_msgs
```

Comment:
On Jazzy we do not have wasm32 builds for now, so I think it is ok if you just add it to the list as all other deps.